src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -202,6 +202,11 @@ void VM_Version::initialize() {
     if (FLAG_IS_DEFAULT(UseSIMDForMemoryOps)) {
       FLAG_SET_DEFAULT(UseSIMDForMemoryOps, true);
     }
+
+    if (FLAG_IS_DEFAULT(OnSpinWaitInst) && FLAG_IS_DEFAULT(OnSpinWaitInstCount)) {
```

Comment:
Should these two be set independently? If I pass `-XX:OnSpinWaitInstCount=2` then `OnSpinWaitInst` will default to "none".

Reply:
Hi Nick,
Thank you for reviewing the PR.

> Should these two be set independently?

I don't mind.

Reply:
Done.